test/request-processor.js

```diff
@@ -86,4 +89,38 @@ describe('RequestProcessor', () => {
     })
       .then(done, done)
   })
+
+  it('should properly audit every event initiation and termination', (done) => {
+    factory.RequestProcessor.configure({
+      requestImpl: {
+        post: (calledUri, options, cb) => {
+          const response = { calledUri, options }
+          return cb(null, response)
+        }
+      }
+    })
+
+    logger.configure({
+      logImpl: (message) => {
+        assert.ok(typeof message === 'string')
+        assert.ok(/\[LOG\]/.test(message))
```

Comment:
how do we assert this code even runs? ie: right now, if nothing is ever logged, this test passes.

Reply:
Good point. I'll see if I can set up a spy to ensure the method is running.